### DISCUSSION TITLE
Always defined value from useQueries with initialData

### DISCUSSION THREAD

**Discussion by nik-webdevelop** [TanStack/query#6278] [Q&A] [answered]
Is there any chance to see in the future always defined `useQueries` result when `initialData` is set ? (the same as in `useQuery`)

```
const [{ data }] = useQueries({
    queries: [
      {
        ...getTodoQueryOptions(),
        initialData: initialTodo,
      },
    ],
  });
```

`data` is `Todo | undefined`

**Comment by TkDodo** [comment, marked as answer]
not on my immediate list of things to work on, sorry. If you're a TS wizard 🧙 , have a go at it :)

**Reply by nik-webdevelop** [reply]
Ok, thanks )
